elastalert/elastalert.py

```diff
@@ -403,6 +403,9 @@ def set_starttime(self, rule, endtime):
             # If we started using a previous run, don't go past that
             if 'minimum_starttime' in rule and rule['minimum_starttime'] > endtime - buffer_time:
```

Comment:
it would make sense to extract `endtime - buffer_time` to a variable as it is used 3x in the subsequent lines. Call it sth with `_delta` at the end, I guess that would make sense :)